### PR TITLE
Added link to dataprovider for NextJS CRUD

### PR DESCRIPTION
**Pull request by gganebnyi**

### CODE REVIEW

File: docs/Ecosystem.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ See the [translation](./Translation.md#available-locales) page.
 * **[Xmysql](https://github.com/o1lab/xmysql)**: [soaserele/aor-xmysql](https://github.com/soaserele/aor-xmysql)
 * Local JSON: [marmelab/ra-data-fakerest](https://github.com/marmelab/ra-data-fakerest)
 * Simple REST: [marmelab/ra-data-simple-rest](https://github.com/marmelab/ra-data-simple-rest).
+* NestJS CRUD: [FusionWorks/react-admin-nestjsx-crud-dataprovider](https://github.com/FusionWorks/react-admin-nestjsx-crud-dataprovider)
```
Comment (djhi) [inline docs/Ecosystem.md]: Would you mind adding it like the others (format) and in alphabetical order ? (The local and simple ones are exceptions here). Also please add it to the `DataProviders.md` page too. We still have to decide which one we'll keep...

Comment (gganebnyi) [reply docs/Ecosystem.md]: @djhi updated!

**Comment by fzaninotto** [comment]
Thanks!

**Comment by Bnaya** [comment]
amazing thank you

**Pull request merged**